index.android.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
we should probably just have one entry file

Reply:
Got it... That's fine. I think that's just RN convention

Reply:
Lets just make one entry file that should work for both: `index.js`